shlr/tcc/tccgen.c

```diff
@@ -1792,8 +1790,7 @@ ST_FUNC void indir(void)
 
 /* parse an expression of the form '(type)' or '(expr)' and return its
    type */
-static void parse_expr_type(CType *type)
-{
+static void parse_expr_type(CType *type) {
```

Comment:
Also inconsistency here - why do you need to have `{` at the same line?

Reply:
because this is not TCC anymore, so it should follow our coding style